pkgs/development/tools/continuous-integration/github-runner/default.nix

```diff
@@ -233,7 +200,6 @@ stdenv.mkDerivation rec {
     install -m755 src/Misc/layoutroot/env.sh                   $out/lib/
 
     # Rewrite reference in helper scripts from bin/ to lib/
-    substituteInPlace $out/lib/run.sh    --replace '"$DIR"/bin' '"$DIR"/lib'
```

Comment:
This removal should be the only functional change to the package. I removed the line because it complained `"$DIR"/bin` doesn't exist in that file.